### DISCUSSION TITLE
🔗 Allow the user to choose the image size for an ACF background image (Duplicate of #22750)

### DISCUSSION THREAD

**Discussion by hein-obox** [elementor/elementor#22801] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When I select a background image from the media library I can select the image size (e.g. Full, Thumbnail, Medium) etc. 
But I select the background image from an ACF Field I don't have this option.
I would like to have the image size options for ACF Fields. 

The same applies for Featured Images. I can select the size of the Featured Images.

### Describe the solution you'd like

See above.

**Comment by hein-obox** [comment]
I realised this is the same issue as described here: https://github.com/elementor/elementor/issues/22055. And here: https://github.com/orgs/elementor/discussions/22750.

**Comment by nicholaszein** [comment]
Hey @hein-obox! 🙂 

I'm merging this request with #22750 which is on the same subject! 😉
